include/micm/process/cuda_process_set.cuh

```diff
@@ -21,5 +21,22 @@ namespace micm
         int product_ids_size,
         const double* yields,
         int yields_size);
+
+    void AddJacobianTerms_kernelSetup(
+        const double* rate_constants, 
+        const double* state_variables, 
+        size_t n_grids, 
+        size_t n_reactions, 
+        size_t n_species,
```

Comment:
In the `AddForcingTerms_kernelSetup` function, different variable names and types are used. Either way works for me but I would suggest using the consistent declaration.

Reply:
I updated the variable names and data type the same in both AddJacobianTerms and AddForcingTerms for consistency.